ui/packages/shared/profile/src/IcicleGraph.tsx

```diff
@@ -49,6 +54,10 @@ interface IcicleGraphNodesProps {
 
 interface IcicleGraphRootNodeProps {
   node: FlamegraphRootNode;
+  strings: string[];
```

Comment:
As a future improvement, we can create a `FramegraphContext` and access the tables through that so that we don't have to pass them as props to every component.

Reply:
That would be amazing! I'll create an issue to track it.